.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: false
```

Comment:
Huh, I wonder if this is different for collaborators vs regular users? The https://github.com/iree-org/iree/issues/new link works as expected for me...

Here's what I see:
![image](https://user-images.githubusercontent.com/4010439/186533166-976a35e6-dc54-4dec-9df7-afea077201e5.png)

click "Open" on the third list item (Blank Issue):
![image](https://user-images.githubusercontent.com/4010439/186533199-96b1c7dd-14eb-4d13-b417-2298ead9ac8c.png)

Reply:
Interesting, that may be the case. If so only allowing collaborators to create blank issues may be preferred? Happy to close this pull if so.

For the sake of a paper trail, here is my 302 redirect to the choose page:

<img width="693" alt="image" src="https://user-images.githubusercontent.com/3036970/186533897-35366467-ffa2-4ea1-8c42-93adca5b08c1.png">

Reply:
We went back and forth a few times trying to make "create a blank issue" more prominent: https://github.com/iree-org/iree/pull/9860

We had many project members using the templates for tracking tasks that would have been better as blank issues. Having just the text at the bottom was easy to miss.

That redirect is annoying though... we don't want to present a broken (redirect loop) UI to contributors...

Reply:
I wonder if it's actually only repo _admins_ that can still create blank issues when that is disabled...
@pzread or someone else who doesn't have admin, can you try to create a Blank Issue from https://github.com/iree-org/iree/issues/new/choose and report if there's a redirect?

Reply:
What is the URL for the blank issue link in the bottom if you enable that?

Reply:
Further evidence: https://github.com/community/community/discussions/14287. That is so totally not what I would expect to happen. I think it's fine to have `blank_issues_enabled` set to true if we want non-collaborators to be able to open blank issues. It just adds a little link at the bottom

Reply:
Referring to the "Don't see your issue" link at the bottom-left? 

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/3036970/186536438-86c9c81b-afb5-4c82-9a4a-7b64ae6c754d.png">

Image from https://github.com/openxla/stablehlo/issues/new/choose

Reply:
It'll be a bit silly to show "blank issue" in the main section _and_ down at the bottom, but that looks like the best we can do right now :/